Sources/StreamChatUI/UIConfig.swift

```diff
@@ -9,7 +9,7 @@ public typealias UIConfig = _UIConfig<NoExtraData>
 
 public struct _UIConfig<ExtraData: ExtraDataTypes> {
     /// A view used as an online activity indicator (online/offline).
-    public var onlineIndicatorView: UIView.Type = _ChatOnlineIndicatorView<ExtraData>.self
+    public var onlineIndicatorView: (UIView & MaskProviding).Type = _ChatOnlineIndicatorView<ExtraData>.self
```

Comment:
Maybe this can stay `UIView`? There's not reason to force users to provide a mask if the don't need to cut out the superview.

Reply:
I was thinking about this a lot. If we don't request the mask, users might be surprised why the custom view is not masked and could take them a good amount of time to realize how to achieve that from our implementation. By asking them to conform to this protocol, it's explicit and easy to just return nil when they don't want to use the mask.